src/WinClient.cc

```diff
@@ -319,7 +324,14 @@ void WinClient::updateTitle() {
     if (m_title_override)
         return;
 
-    m_title.setLogical(FbTk::FbString(Xutil::getWMName(window()), 0, 512));
+    FbTk::FbString fullname = FbTk::FbString(Xutil::getWMName(window()), 0, 512);
+    FbTk::FbString clientmachine = FbTk::FbString(Xutil::getWMClientMachine(window()), 0, 512);
+    char *host = new char[512]; gethostname(host, 512);
```

Comment:
nope.

just char host[512];

you are memory-leaking yourself here.

Reply:
fixed in the latest commit!

Reply:
i did not see it fixed yet, in which commit did you fix it?